node/narwhal/src/helpers/cache.rs

```diff
@@ -12,33 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use snarkvm::{console::types::Field, ledger::narwhal::TransmissionID, prelude::Network};
-
 use core::hash::Hash;
-use indexmap::IndexMap;
-use parking_lot::RwLock;
 use std::{
-    collections::VecDeque,
+    collections::{BTreeMap, HashMap},
     net::{IpAddr, SocketAddr},
 };
+
+use parking_lot::RwLock;
+use snarkvm::{console::types::Field, ledger::narwhal::TransmissionID, prelude::Network};
 use time::{Duration, OffsetDateTime};
```

Comment:
nit: incorrect ordering of import declarations

Reply:
fixed